roles/build-image/files/foreman/container-assets/start_foreman.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
I think this is an example of some clean up that is needed after you split up to `foreman-dockerfile` and `foreman-proxy-dockerfile`.

Reply:
Good catch, removed.